tests/testsuite/install.rs

```diff
@@ -2623,3 +2623,598 @@ fn uninstall_running_binary() {
         )
         .run();
 }
+
+#[cargo_test(nightly, reason = "--dry-run is unstable")]
+fn dry_run_freshinstall_one() {
```

Comment:
Please add these tests in their own commit at the start, without the dry-run flag, so we can compare what the dry-run flag does to the output

Reply:
Just to be clear, you want a copy of each test where only the flag is removed and the output adapted in consequence, right? Also, when you say "add these tests in their own commit at the start", are you referring to the already-written tests, the "copies" that need writing or all of them?

Reply:
Each test that exists in this PR should be added in their own commit, the first one.  The `--dry-run` flag would be missing.  These tests would pass.

The next commit in the PR would update the tests to add `--dry-run` and update the output accordingly.

Reply:
OK, I see.